packages/dx-react-grid/docs/reference/virtual-table.md

```diff
@@ -17,6 +17,19 @@ estimatedRowHeight | number | `37` for [Bootstrap3](https://www.npmjs.com/packag
 
 This plugin also supports the [Table](table.md#properties) plugin's properties.
 
+## Plugin Components
```

Comment:
I don't like an idea to duplicate docs for VirtualTable. Can we avoid it?

Reply:
@kvet  Maybe we must write the next sentence?
"You can use all Table's components from VirtualTable. For example VirtualTable.Cell"

Reply:
Yes, it's way better. @viterobk what do you think?

Reply:
I'd better duplicate all the Table plugin's descriptions in Virtual Table topic. "Same as Table's" could be appropriate if Virtual Table actually extended Table. However, it simply has similar properties, dependencies, imports an exports.